openapi-generator/lib/src/openapi_generator_runner.dart

```diff
@@ -493,4 +494,36 @@ class OpenapiGenerator extends GeneratorForAnnotation<annots.Openapi> {
       );
     }
   }
+
+  /// Format the generated code in the output directory.
+  Future<void> formatCode({required GeneratorArguments args}) async {
+    final command = Command(executable: 'dart', arguments: ['format', './']);
+    ProcessResult result;
+    if (!testMode) {
+      result = await Process.run(
```

Comment:
Please deprecate this testMode flag. It will be removed in the future. We shouldn't be using flags to determine if running in test environment or not. Mocks are preferable.

See [test_process](https://pub.dev/packages/test_process)

Reply:
🤔 It doesn't seem like it is able to mock the instance of the process (since it isn't something that is passed to a function). I could do a refactor that individually tests the function calls but that lift would be out of scope (though I could have a bunch of that up by tonight). This will also remove that flag too. Would you mind that for this PR the test mode is left?

Reply:
Yeah. sure.